archaius2-archaius1-bridge/src/main/java/com/netflix/archaius/bridge/BackportingConfigProxyFactory.java

```diff
@@ -0,0 +1,21 @@
+package com.netflix.archaius.bridge;
+
+import com.netflix.archaius.ConfigProxyFactory;
+import com.netflix.archaius.api.Config;
+import com.netflix.config.ConfigurationManager;
+
+public class BackportingConfigProxyFactory extends ConfigProxyFactory {
+
+    public BackportingConfigProxyFactory(Config config) {
+        super(config);
+    }
+    
+    @Override
+    protected <T> MethodInvoker<T> createDynamicProperty(Class<T> type, String propName, String defaultValue) {
+        if(defaultValue != null && getConfig().getRawProperty(propName) == null)
```

Comment:
Not a typo, I'm checking to see if the property is not already set from Archaius1. I could be more explicity about that and just use the static method if that helps. Normally I'd set this at the DefaultLayer but since we don't have that concept when backporting I'm just performing this check. Open to other ideas on how to accomplish this.